packages/extension-ui/src/Popup/CreateAccount/SafetyFirst.tsx

```diff
@@ -25,23 +26,36 @@ const WrapperRow = styled.div`
   width: 100%;
 `;
 
+const StyledFooter = styled(HelperFooter)`
+  .icon {
+    margin-top: -12px;
+  }
```

Comment:
There's no way to move some top/bottom padding/margin to lower value rather than using minus values on icon?